### ISSUE TITLE
Turning in Quests?

### ISSUE THREAD

**Issue by Shogunofnight** [AeroScripts/QuestieDev#211]
Interface/AddOns/!Questie/Questie.lua:967: attempt to concatenate local `name' (a nil value)

This message keeps popping up in chat every time I try and press the complete quest button to turn it in. I don't know what to do about it besides logging out turning off questie then turning it in. Any ideas?

**Comment by Dyaxler** [comment]
That line refers to a VERY old function that has been around since before I started making changes. I haven't touched that code. If it's error'ing out then it's probably falling back to this legacy function due to one of the newer ones failing to parse the quest name upon turn-in. Questie was written for Nostalrius and their method of returning quests is different on other servers. For example, the server I'm on now prevents the quests level from being changed by mods in the quest description panel. I've had to build in a check to see if the quest level is present in the quest name. Questie relies on the title of the quest that is in the players quest log. There might be something else that's being inserted into the quest title on your server?

Which version of Questie are you using and which server are you playing on?

Have you tried running **/questie clearconfig** in game?
